README.md

```diff
@@ -1 +1,53 @@
-# stephen-tribute-page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
[OPTIONAL] 

- [ ] you can add a screenshot of your webpage to let know your visitors what the page looks like.